File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        // brute force O(n) time and space
        // copy twice

        vector<int> ans = nums;
        for (int i = 0; i < nums.size(); i++) {
            ans.push_back(nums[i]);
        }

        return ans;
    }
};

File: Data Structures & Algorithms/single-number/submission-0.py
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        # brute force (O(n) time and space)
        freq = {}
        for num in nums:
            if num in freq:
                freq[num] = 2
            else: 
                freq[num] = 1
        
        for k,v in freq.items():
            print(str(k) + " " + str(v))
            if v == 1: 
                return k
        
        return -1
    